### PR TITLE
feat(frontend): add Frontend and README

### PR DESCRIPTION
**Pull request by ZaiKuku**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+1. Open the terminal and navigate to the project's root directory.
+2. Run the following command to install project dependencies:
+
+   ```bash
+   npm i
+   ```
+
+   This will download and install all the required resources for the project.
+
+### Start the Backend
```
Comment (yenting-biao) [inline README.md:20]: backend 是用 yarn，然後應該是
```
$ cd backend
$ yarn
$ yarn dev
```

**Comment by ZaiKuku** [comment]
我改成yarn了

File: backend/package-lock.json
Comment (togimoto) [inline backend/package-lock.json:1]: Please remove this file from the PR.

File: backend/package.json
Comment (togimoto) [inline backend/package.json:1]: Please remove this edit from the PR.

File: backend/yarn.lock
Comment (togimoto) [inline backend/yarn.lock:1]: Please remove this edit from the PR.

**Review by togimoto — DISMISSED** [review]
Please remove any changes to /backend/ from the PR. Backend team will handle the backend section of README.

**Comment by ZaiKuku** [comment]
新增資料庫設定的說明

**Review by quevenliu — APPROVED** [review]

**Pull request merged**